fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1898,14 +1898,10 @@ public int getAsInt() {
                 modifyFieldColumns = Set.of(addFieldClause.getColName());
                 checkModifiedColumWithMaterializedViews(olapTable, modifyFieldColumns);
 
-                Locker locker = new Locker();
-                locker.lockDatabase(db, LockType.READ);
-                int id = 0;
-                try {
-                    id = olapTable.incAndGetMaxColUniqueId();
-                } finally {
-                    locker.unLockDatabase(db, LockType.READ);
-                }
+                // NOTE
+                // Here is already hold the write lock of database (AlterJobMgr::processAlterTable)
```

Comment:
So that means this function can only be called by `AlterJobMgr::processAlterTable`?
But my question is that if `AlterJobMgr::processAlterTable` is refactored someday, will it cause new issues here? Because he/she may not know this function depends on its implementation. It seems there is no statement or function name to notice developer that this function needs db lock to be held.